Makefile

```
@@ -1,3 +1,11 @@
 .PHONY: build
 build:
-	go build -ldflags "-s -w -X 'github.com/friendsofgo/killgrave/internal/app/cmd._version=`git rev-parse --abbrev-ref HEAD`-`git rev-parse --short HEAD`'" -o bin/killgrave cmd/killgrave/main.go
\ No newline at end of file
+	go build -ldflags "-s -w -X 'github.com/friendsofgo/killgrave/internal/app/cmd._version=`git rev-parse --abbrev-ref HEAD`-`git rev-parse --short HEAD`'" -o bin/killgrave cmd/killgrave/main.go
+
+.PHONY: build-docker
+build-docker:
+	docker build --build-arg TAG=$(TAG) -t killgrave:$(TAG) .
+
+.PHONY: test
+test:
+	go test -v -vet=off -race ./...
```

```suggestion
	go test -v -count=1 -vet=off -race ./...
```

I'd suggest adding `-count=1` to prevent caching, and thus make results more reliable at almost no cost (as the suite is pretty fast).

Additionally, could you explain why do we need `-vet=off`, please?

---

I see it's also on the CI, so perhaps an opportunity to remove it from there? Unless we have a reason to keep it, of course. But I cannot see it and I don't remember any.